test-data/unit/check-flags.test

```diff
@@ -482,6 +482,25 @@ def f() -> None:
     x = [] # type: Union[List[Optional[str]], str]
 [builtins fixtures/list.pyi]
 
+[case testPerFileStrictOptionalNoneArguments]
+# flags: --config-file tmp/mypy.ini
+import standard, optional
+
+[file standard.py]
+def f(x: int = None) -> None: pass
+[file optional.py]
+import standard
+def f(x: int = None) -> None: pass
+standard.f(None)
+
+[file mypy.ini]
+[[mypy]
+strict_optional = False
+[[mypy-optional]
+strict_optional = True
+
```

Comment:
Fixed the three blank lines.  The lines between the files make them much more readable to me (though it looks like one got dropped, so I added it back).